packages/react/src/hooks/actions/testShared.ts

```diff
@@ -1,26 +0,0 @@
-import type { ModelInit, MutableModel } from '@aws-amplify/datastore';
```

Comment:
this file is used in action hooks

Reply:
ah gotcha, moved this so it is with the other test models